components/paragraph/src/test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from 'enzyme';
+
+import Paragraph from './';
+import LinkRenderer from './link-renderer';
```

Comment:
we should consider splitting the `LinkRenderer` tests out to a separate test file

Reply:
Definitely, thanks for this and the ticket!